packages/react-scripts/template/README.md

```diff
@@ -1479,6 +1480,67 @@ If you use [Visual Studio Code](https://code.visualstudio.com), there is a [Jest
 
 ![VS Code Jest Preview](https://cloud.githubusercontent.com/assets/49038/20795349/a032308a-b7c8-11e6-9b34-7eeac781003f.png)
 
+## Debugging Tests
+
+There are various ways to setup a debugger for your Jest tests. We cover debugging in Chrome and Visual Studio Code.
+
+### Chrome
+
+Place `debugger;` statements in any test and run
+
+macOS/Linux
+```bash
+node node_modules/.bin/react-scripts --inspect-brk test --runInBand --env=jsdom
+```
+
+Windows
+```bash
+node node_modules/bin/react-scripts --inspect-brk test --runInBand --testPathIgnorePatterns=ui-tests/* --env=jsdom
+```
```

Comment:
On a general note: instead of a Terminal command, can we suggest adding

```
  "test:debug": "react-scripts --inspect-brk test --runInBand --env=jsdom"
```

to `scripts` in `package.json` and then ask you to run `npm run test:debug`?

Reply:
Yeah, that is much cleaner. Updated!